services/ymax-planner/src/engine.ts

```diff
@@ -418,8 +429,69 @@ export const pickBalance = (
   );
 };
 
+/**
+ * Process pending transactions based on their individual age
+ */
+export const processInitialPendingTransactions = async (
+  initialPendingTxData: PendingTxRecord[],
+  rpc: CosmosRPCClient,
+  txPowers: HandlePendingTxOpts,
+  now: () => number,
+  handlePendingTxFn = handlePendingTx,
+) => {
+  if (initialPendingTxData.length === 0) return;
+  const { log = console.warn.bind(console) } = txPowers;
```

Comment:
`startEngine` has access to the ambient console (for now), but that is not appropriate for any other function in this file.
```suggestion
  const { log = () => {} } = txPowers;
```